src/main/java/com/diffmin/util/SpoonUtil.java

```diff
@@ -73,19 +75,27 @@ public static CtModel buildModel(File file) throws FileNotFoundException {
         return launcher.buildModel();
     }
 
-    /**
-     * Pretty prints the model.
-     *
-     * @param model model to be pretty printed
-     * @return patched program
-     */
```

Comment:
Why remove the documentation? Also, I think it's important to mention in the javadoc that the model is expected to have only one compilation unit.

Reply:
I was shifting this method to `StructureTest` but then I realised it is used elsewhere too. In that transfer, it would have been removed. I will revert this change.